### PR TITLE
BAH-515 | Add amman bed management module

### PR DESCRIPTION
**Pull request by sai789**
https://bahmni.atlassian.net/browse/BAH-515

We forked your repo and added our files and tested on Bahmni DB. Everything working fine.
Two common files changed as part of this. 
1. `ui/app/common/concept-set/mappers/observationMapper.js`
2. `ui/app/common/domain/services/bedService.js`

In `observationMapper.js` file, removed a unused variable in a method.
In `bedService.js` file, added a method `getCompleteBedDetailsByBedId()`

Here are the changed files list:
app/ipd/ --> new folder
app/i18n/ipd/ --> new folder
app/styles/ipd/ --> new folder
/test/unit/ipd/ --> new folder
karma.conf.js --> Include ipd tests

### CODE REVIEW

File: ui/app/i18n/ipd/locale_pt_BR.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (djazayeri) [inline ui/app/i18n/ipd/locale_pt_BR.json]: translations should not be merged in the main product, but these should happen in transifex (like in other apps)

Comment (sai789) [reply ui/app/i18n/ipd/locale_pt_BR.json]: We removed this translation file in PR

File: ui/app/common/concept-set/mappers/observationMapper.js [hunk 2]
```diff
@@ -54,12 +54,12 @@ Bahmni.ConceptSet.ObservationMapper = function () {
         });
     };
 
-    var mapObservation = function (concept, savedObs, conceptSetConfig, parentConcept) {
```
[the remark this answers is not in the document]
Comment (sai789) [reply ui/app/common/concept-set/mappers/observationMapper.js]: We have removed extra parameter but we forgot to remove in method calls. Now we removed extra passing parameter wherever it is called.

**Comment by sai789** [comment]
@SwathiVarkala We have addressed these feedbacks. Please have a look at them
1. Gender display in bed management (restricted to M/F) : ipd/bedManagement.html
2. i18 is not done for Gender field on bed management page ipd/bedManagement.html
3. camelCase not used for method, variable names

**Comment by sai789** [comment]
@SwathiVarkala We thought to implement your suggestion. But it seems the createLayoutGrid function implemented with two forLoops which are creating required empty beds, but with this approach we are unable to generate the empty beds.
1. bedManagementService.js createLayoutGrid can be simplified using the below object sort [{x:1,y:2},{x:2,y:1},{x:1,y:3}].sort((a,b) => parseInt(a.x+''+a.y) - parseInt(b.x+''+b.y)) sorting

**Pull request merged**